### DISCUSSION TITLE
Welcome to judo_rest_api Discussions!

### DISCUSSION THREAD

**Discussion by OStrama** [OStrama/judo_rest_api#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OverDodo** [comment]
Hey there, just wanted to say thank you for working on this! The WebApi is not working for me anymore so I started looking around and was super happy when I found your post. Really excited for this! 🙂

**Reply by OStrama** [reply]
You are welcome! Nice to see that there‘s interest out there on this integration. 
After Christmas I hope that I‘ll find some time to continue it. I prepared integration of Number and Switch entities and also calculated entities to calculate l/h from water consumption. 
However, I faced a challenge regarding the leakage protection: There are REST addresses to close and open it, but no address to check the current state? That makes it hard to implement the leakage protection as a switch, especially as it can be closed automatically in vacation mode..
I think I have to get in contact with Judo and hope that they have some ideas and hints…

**Reply by OverDodo** [reply]
Yea I noticed there is a ton of data missing compared to the web api. Which is just annoying. I was hoping they would at least have some sort of Status endpoint. The rest api also seems to be missing reading values like maximum flow etc.. Though these readings arent that important for everyday use. What i'd really miss is the current hardness of the water coming in, which they also dont seem to provide through rest.. Unless there is endpoints they dont mention in their sheet?

**Reply by OStrama** [reply]
I hope that there are undocumented endpoints. But scanning is dangerous, since it can change parameters that I do not know..

**Comment by oliverbiegel** [comment]
Hi,

Thanks a lot for working on this solution! I have downloaded the respository in Home Assistant but however I don't know how to get to the config window that is mentioned in the description...I am still kind of a beginner - can you please guide me on that?

**Comment by OStrama** [comment]
You should get it as follows: After adding the repository to HACS and a restart of home assistant (as far as I remember?) you should open "Devices and Services" or "Geräte und Dienste" in "Settings" or "Einstellungen":
![image](https://github.com/user-attachments/assets/fe555b56-4abf-4ab5-b1e7-49e7d9d0eb20)
Then press "Add integration" or "Integration hinzufügen" bottom right:
![image](https://github.com/user-attachments/assets/8ef8f367-ebd7-4661-b795-a644c053c933)
Then search for "Judo" and doubleclick the entry:
![image](https://github.com/user-attachments/assets/9cca5ed3-404d-4426-b7a8-2c13c1aebc2d)

**Reply by oliverbiegel** [reply]
Thanks for the answer (I just missed the logical last step)!

**Comment by OStrama** [comment]
You are welcome!
